week_2/dagster_ucr/project/week_2.py

```diff
@@ -5,26 +5,53 @@
 from dagster_ucr.resources import mock_s3_resource, redis_resource, s3_resource
 
 
-@op
-def get_s3_data():
-    pass
+@op(
+    config_schema={"s3_key": str},
```

Comment:
I didn't add this as a parameter in my code, thinking that it was redundant given that it's set in code in the local and docker configs and can be referenced using `context`. But I noticed that Dennis uses it in the ETL example, and now I'm curious about best practices. Have you seen this discussed or if you have thoughts? 🤔